src/bootstrap3/renderers.py

```diff
@@ -311,6 +313,15 @@ def add_widget_attrs(self):
 
     def list_to_class(self, html, klass):
         classes = add_css_class(klass, self.get_size_class())
+        if IS_PRE_DJANGO4:
+            return self._list_to_class_pre_django4(html, classes)
+        return self._list_to_class(html, classes)
+
+    def _list_to_class(self, html, classes):
+        html = re.sub("<div>\s*<label", f'<div class="{classes}"><label', html)
+        return html
```

Comment:
I'd suggest this, but feel free to ignore:

```suggestion
        return re.sub("<div>\s*<label", f'<div class="{classes}"><label', html)
```

Reply:
Done